ISSUE_TEMPLATE

```diff
@@ -0,0 +1,17 @@
+# Issue Title
+
+Provide a descriptive title.
+
+# Version info 
+
+Provide the .NET Framework Release key value and your Windows version (output of `winver`).
+
+HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\NET Framework Setup\NDP\v4\Full, Release 

```

Comment:
In [How to: Determine Which .NET Framework Versions Are Installed](https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/how-to-determine-which-versions-are-installed#net_b), we document the path as HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\NET Framework Setup\NDP\v4\Full, Release. Are they the same?